docker/python3-scientific/Dockerfile

```diff
@@ -6,11 +6,13 @@ LABEL maintainer="Ambrose J. Carr <acarr@broadinstitute.org>" \
 
 RUN pip3 install \
     crimson==0.3.0 \
+	HTSeq==0.9.0 \
     matplotlib==2.1.0 \
     numpy==1.12.0 \
     pandas==0.20.3 \
     pysam==0.12.0.1 \
-    requests==2.18.4
+    requests==2.18.4 \
     scsequtil==0.0.6 \
     scipy==0.18.1 \
-    tables==3.4.2
+    tables==3.4.2 
+
```

Comment:
Could you tell me with which tag you pushed this image? I saw all images has been updated on https://hub.docker.com/r/humancellatlas/python3-scientific/tags/